.github/workflows/create_discussion.yml

```
@@ -0,0 +1,126 @@
+# @usage
+#
+# on:
+#   schedule:
+#     - cron: "0 0 * * mon"
+# jobs:
+#   create_discussion:
+#     uses: route06/actions/.github/workflows/create_discussion.yml@v2
+#     with:
+#         target_day: sunday
+#         owner: route06
+#         repository_name: professional-service-div-template
+#         title: ハドル
+#         discussion_category_slug: meeting-notes
+#         description_template_path: _templates/weekly_meeting_discussion/test.md
+
+name: Create discussion
+
+on:
+  workflow_call:
+   inputs: 
+    target_day:
+      required: true
+      type: string
+    owner:
+      required: true
+      type: string
+    repository_name:
+      required: true
+      type: string
+    title:
+      required: true
+      type: string
+    discussion_category_slug:
+      required: true
+      type: string
+    description_template_path:
+      required: true
+      type: string
```

https://github.com/route06/actions/pull/52/commits/7101e9e3cc06728d29eb6d0825a26c7a6014f88d#diff-2e3d055b3d4077c5c9d733ef45c46fca4629491e1664ade5a55c4d2bb9e20b7fL18-L27
この時の説明、かなり優しさがあり良いなと思っていました！

提案: それぞれdescriptionフィールドを足して説明をすると良さそうでしたがどうでしょうか？

```suggestion
      type: string
      description: 作成するDiscussionの説明文に利用するテンプレートファイルのパスを設定してください
```

---

以下にて対応しました！
041807d6cfef5a53c9c8e7c19e0334ecf9cc40a4